libs/cuda_support/tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if (NOT HPX_WITH_CUDA_SUPPORT)
+  return()
+endif()
+
+#enable_language(CUDA)
```

Comment:
Is this not needed anymore?

Reply:
It wasn't there originally. I added it then commented it out. Gone now.